calblueprint/meepanyar
# Add financialSummaries patch and patch command

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
This PR adds a new patch to the set of patches and also creates an npm command `npm run apply-schema-patches` that'll apply schema patches.

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review
- Run `npm run generate-schema` and look at the diff via git status. The diff should show that the `schema.js` file was modified. 
- Now run `npm run apply-schema-patches` and the new diff should be clean.
- Ensure that the `financialSummaries` custom object is still located under the `Sites` object

![image](https://user-images.githubusercontent.com/35501399/109913134-3f906900-7c62-11eb-8c7d-84f7104df88a.png)


[//]: # 'The order in which to review files and what to expect when testing locally'

## Relevant Links

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
